lang/cs/Org.Apache.REEF.IMRU.Tests/MapperCountTest.cs

```diff
@@ -30,7 +30,7 @@ namespace Org.Apache.REEF.IMRU.Tests
     [TestClass]
     public class MapperCountTest
     {
-        private const int NumberOfMappers = 7;
+        private const int NumberOfMappers = 1;
```

Comment:
Why this change? I deliberately choose a prime number here to unearth any potential assumptions in the code.